language/en-GB/en-GB.com_users.ini

```diff
@@ -26,6 +26,7 @@ COM_USERS_EMAIL_REGISTERED_WITH_ADMIN_ACTIVATION_BODY_NOPW="Hello %s,\n\nThank y
 COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your %s account.\n\nYour username is %s.\n\nTo login to your account, select the link below.\n\n%s \n\nThank you."
 COM_USERS_EMAIL_USERNAME_REMINDER_SUBJECT="Your %s username"
 COM_USERS_ERROR_SECRET_CODE_WITHOUT_TFA="You have entered a Secret Code but two factor authentication is not enabled in your user account. If you want to use a secret code to secure your login please edit your user profile and enable two factor authentication."
+COM_USERS_ERROR_USER_NOT_FOUND="User not found"
```

Comment:
Strings need to be alpha ordered.

Reply:
They are ordered. My key is with ERROR not with EMAIL so I think the order is correct.

Reply:
Ah sorry, you are right.